www/json/trip_confirm_options.json.sample

```diff
@@ -1,25 +1,41 @@
 {
+
   "modeOptions" : [
-       {"text":"Walk", "value":"walk"},
```

Comment:
To clarify, you need to modify `trip_confirm_options.json` and leave `trip_confirm_options.json.sample` unchanged to avoid merge conflicts.